scripts/globals/utils.js

```
@@ -1,4 +1,7 @@
 import angular from 'angular';
+import moment from 'moment/min/moment-with-locales';
```

Maybe you don't see that but this weird import doesn't have type definitions so it's highlighted as an error.
Could add:
```
// @ts-ignore
```
above it to silence it.

---

I don't see this. But I added the comment. According to the web, it's customary to add a description of the silenced error. I hope, this still works now.

---

It would be nice to think about locales for Angular, perhaps.

---

You mean sth like that?
https://www.npmjs.com/package/angular-timeago

I guess, the footprint will be much smaller. It supports de, en, pl, but not (yet) russian. Angular itself doesn't seem to have a localized `timeAgo` module...

---

No, I mean locales for Angular in general as it looks like a lot of people want to localise TB into their language. Having all localities would be a an overkill, but perhaps de, es, fr, ru, pl would be nice to have out of the box.

---

If I understand the docs right, localizing angular is simply a matter of including the angular localization file, e.g. `i18n/angular-locale_de-de.js`.
https://docs.angularjs.org/guide/i18n

Similar things seem to hold for moment locales, e.g. `locale/fr.js`. Plus we need to call `moment.locale('fr')`.
https://momentjs.com/docs/#/i18n/loading-into-browser/

So, we could
* provide `cat`'ed locales for both modules in some `locales` folder
* include the `moment.locale` call in those files
* provide a `utils.js` function to load those files from `config.js`, e.g. this: https://github.com/resoai/TileBoard/wiki/Splitting-the-configuration
* add info on how to use in the docs.

I'll try, if this works. If it does, I'll leave out the locales from the bundle for the present PR. 

We can then work on the localization separately.

---

Yes, that works. With the recent commits, the following will work in `config.js` (assuming that we're living off the `build` directory):
```
loadScript('../node_modules/angular-i18n/angular-locale_de-de.js');
loadScript('../node_modules/moment/locale/de.js');
```

I'd now like to bundle the localization files using rollup, as this will allow us to catch any updates. But I have no clue, how this should be done...  :-/

---

Sorry... I had promised to make i18n a separate PR. :-) So, this one is a very simple now...